torch/distributions/transforms.py

```diff
@@ -376,8 +392,11 @@ def _inverse(self, y):
         return (y - self.loc) / self.scale
 
     def log_abs_det_jacobian(self, x, y):
-        result = torch.abs(self.scale).log()
         shape = x.shape
+        scale = self.scale
+        if isinstance(scale, numbers.Number):
+            scale = x.new_empty(shape).fill_(scale)
+        result = torch.abs(scale).log()
```

Comment:
Minor optimization: abs can be lifted to the scalar case and would be cheaper

Reply:
Good point, I'll move the `log` in there too.